examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2153,6 +2190,78 @@ impl FrontendNode {
         Ok(())
     }
 
+    fn test_http_server_create(&mut self) -> TestResult {
+        // Create an HTTP server pseudo-Node.
+        let result = oak::http::init("[::]:8282");
```

Comment:
Is there any reason why this particular test uses a different port from the others? Either way, maybe add a comment; alternatively, make the fixed address a const?